README.md

```diff
@@ -35,6 +35,11 @@ sudo apt-get install git
 sudo apt-get install openjdk-7-jdk
 sudo apt-get install npm
 sudo apt-get install libfontconfig
+sudo npm -g install grunt-cli
```

Comment:
Thanks!  I totally agree it would be better to have these install automatically--I'll look and see if I can modify the `pom.xml` to get that working without the additional steps.